content/applications/marketing/sms_marketing/pricing/pricing_and_faq.rst

```diff
@@ -31,106 +30,81 @@ Limit per SMS: 70 characters.
-   test the feature.
-
+   If you are on Odoo Online (SAAS) and have the Enterprise version, you benefit from free trial
+   credits to test the feature.
 
 More common questions
 =====================
 
-#. **Is there an expiration time for my credits?**
-
-   No, credits do not expire.
-
-
-#. **Can I send an SMS to a phone number (which is not a mobile phone) because I see the icon in
-   front of the field “phone”?**
-
-   Only if that phone number supports SMS (e.g. SIP phones).
-
-
-#. **Do I receive an invoice to buy my credits?**
-
-   Yes.
-
-
-#. **Can the recipient answer to me?**
-
-   No, it is not possible to reply to the SMS.
-
-
-#. **What happens if I send multiple SMS but I do not have enough credits to send them all?**
-
-   The whole transaction is counted as a single one, so no SMS will be sent until you have enough
-   credits to send them all.
-
-
-#. **Do I have a history of the sent SMSs?**
-
-   Yes, the SMS is logged as a note under the chatter of the corresponding record, and a complete
-   history of the SMSs sent is available on :ref:`developer mode <developer-mode>` under
-   :menuselection:`Technical --> SMS`.
-
-
-#. **Can I send as many SMSs I want at once?**
-
-   Yes, if you have enough credits.
-
-
-#. **If I have a number that does not exist in the list of recipients, will I lose credits?**
-
-   Not if the phone number is in the wrong format (e.g. too many digits). Otherwise, if the SMS is
-   sent to the wrong person or to a fake number, the credit will be lost.
+#. | **Is there an expiration time for my credits?**
+   | No, credits do not expire.
 
+#. | **Can I send an SMS to a phone number (which is not a mobile phone) because I see the icon in
+     front of the field “phone”?**
+   | Only if that phone number supports SMS (e.g. SIP phones).
 
-#. **What happens if I send my SMS to a paying number (e.g.: a contest to win a ticket for a festival)?**
+#. | **Do I receive an invoice to buy my credits?**
+   | Yes.
 
-   The SMS will not be delivered to that kind of number, so you won’t be charged.
+#. | **Can the recipient answer to me?**
+   | No, it is not possible to reply to the SMS.
 
+#. | **What happens if I send multiple SMS but I do not have enough credits to send them all?**
+   | The whole transaction is counted as a single one, so no SMS will be sent until you have enough
+     credits to send them all.
 
-#. **Can I identify the numbers that do not exist when I send several SMSs?**
+#. | **Do I have a history of the sent SMSs?**
+   | Yes, the SMS is logged as a note under the chatter of the corresponding record. A complete
+     history of SMSs cancelled / queued up / facing delivery errors is available on :ref:`developer
+     mode <developer-mode>` under.
 
-   Only the ones that have an invalid format.
+#. | **Can I send as many SMSs I want at once?**
+   | Yes, if you have enough credits.
 
+#. | **If I have a number that does not exist in the list of recipients, will I lose credits?**
```

Comment:
If an SMS is sent to a number that doesn't exist in the list of recipients, will credits be lost?

No, not if the phone number is incorrectly formatted (e.g. too many digits). However, if the SMS is sent to the wrong person (or to a fake number), the credit for that SMS will be lost.